includes/admin/views/omise-page-card-form-customization.php

```diff
@@ -24,11 +25,19 @@
       <tr>
         <td class="text-bold" style="width: 250px;">Font Name</td>
         <td>
-          <select class="select-input" name="font[name]">
+          <select id="custom_sf_font_name" class="select-input" name="font[name]">
             <option value="Poppins">Poppins</option>
-            <option value="Circular" selected>Circular</option>
+            <option value="Circular">Circular</option>
```

Comment:
Why do we still have references to "Circular"?

Reply:
I guess we never removed it from here.